#object type - box: 1 , cylinder : 2
#object size - box [length,breath,thickness]
#              cyclinder [height,radius] 
#tag_to_object_center - distance from tag to object center in [x, y, z] wrt world
#precedence_constraint - Human action before robot action : 1
#                      - Robot action before human action : 2
#                      - No constraint                    : 3
object_info:
  object_1:
    id: object_1
    tag_id: 1
    type: 1
    dimensions: [0.04, 0.3, 0.07]  
    tag_to_object_center: [0,0,-0.035]
    precedence_constraint: 2
    
  object_2:
    id: object_2
    tag_id: 4
    type: 1
    dimensions: [0.04, 0.3, 0.07]
    tag_to_object_center: [0,0,-0.035]
    precedence_constraint: 2
    
  object_3:
    id: object_3
    tag_id: 8
    type: 1
    dimensions: [0.04, 0.3, 0.07]  
    tag_to_object_center: [0,0,-0.035]
    precedence_constraint: 1

  object_4:
    id: object_4
    tag_id: 10
    type: 1
    dimensions: [0.04, 0.3, 0.07]  
    tag_to_object_center: [0,0,-0.035]
    precedence_constraint: 2